cli/cmd/info/info.go

```diff
@@ -45,10 +52,10 @@ func info(ctx *clicontext.CLIContext) error {
 
 	builder.WriteString(fmt.Sprintf("Rio Version: %s (%s)\n", info.Status.Version, info.Status.GitCommit))
 	builder.WriteString(fmt.Sprintf("Rio CLI Version: %s (%s)\n", version.Version, version.GitCommit))
-	builder.WriteString(fmt.Sprintf("Cluster Domain: %s\n", clusterDomain.Name))
+	builder.WriteString(fmt.Sprintf("Cluster Domain: %v\n", strings.Join(names, ",")))
 	builder.WriteString(fmt.Sprintf("Cluster Domain IPs: %s\n", strings.Join(addresses, ",")))
 	builder.WriteString(fmt.Sprintf("System Namespace: %s\n", info.Status.SystemNamespace))
-	builder.WriteString(fmt.Sprintf("Wildcard certificates: %v\n", clusterDomain.Status.HTTPSSupported))
+	builder.WriteString(fmt.Sprintf("Wildcard certificates: %v\n", strings.Join(certs, ",")))
```

Comment:
This will be a list of booleans ? Should we add additional context so they can see which are true/false ?

Reply:
it kind of matches the order of domains. Not sure what is the best format to convey this.

Reply:
can we loop domains to label: `certificates: domain-A true, domain-B false`